dashboard/controller/users/addUser.js

```diff
@@ -21,7 +21,7 @@ module.exports = function addUser(req, res) {
 		req.body.password = req.body.password.trim();
 		req.body.color = JSON.parse(req.body.color);
 		req.assert('name', common.l10n.get('UsernameInvalid')).matches(regexValidate("user"));
-		req.assert('password', common.l10n.get('PasswordAtLeast', {count:users.ini().security.min_password_size})).len(users.ini().security.min_password_size);
+		req.assert('password', common.l10n.get('PasswordAtLeast', { count: users.ini().security.min_password_size })).len(users.ini().security.min_password_size);
```

Comment:
If changes in the code of this file does not affects the Assignment Feature, then reset the changes in this file.